### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by secretdataz** [SightstoneOfficial/LegendaryClient#254]
LegendaryClient always use 25-35% CPU
Not really lightweight.
![CPU Usage](http://bit.ly/1b1n5Mx)

**Comment by nairaner** [comment]
when does it happen? because for me its 0-1% on main page

**Comment by secretdataz** [comment]
On the main page. My friend also has this issue.

**Comment by nairaner** [comment]
![](http://puu.sh/gw96Z/f4f1933876.png)
For me its 0% 

Can you tell us your specs?

**Comment by secretdataz** [comment]
Mine:
- CPU : Intel Core i5-2410M 2.30GHz
- RAM : 4GB
- OS : Windows 8
- Screenshot in OP

My friend's:
- CPU : Intel Core i5-4690k
- RAM : 32GB
- OS : Windows 8.1
- Screenshot : ![CPU Usage](https://i.imgur.com/HznlopV.png)

Another friend:
- CPU : Intel Core i7-4790K
- RAM : 16GB
- OS : Windows 7
- GPU : NVIDIA GTX770 OC
- Screenshot : ![CPU Usage 2](https://i.imgur.com/P6eGKRf.png)

**Comment by nairaner** [comment]
Can you compare it with cpu usage on login page? It might for some reason still render video

**Comment by secretdataz** [comment]
Login screen
![Loginscreen](http://i.imgur.com/Ucg2rye.jpg)
![Main](http://i.imgur.com/BaC8NJY.png)

**Comment by secretdataz** [comment]
Okay, I think I found the cause of this problem.
Eddy didn't put any code to stop the "Garena sniff" thread after we got all the information we need.
This result in excessive CPU usage. This also explains why the problem doesn't occur on @nairaner because @nairaner doesn't play on Garena.
I'm trying to fix this.

Edit: I can't fix this on first attempt. Maybe I'm wrong. But CPU usage seems to rise after clicking "Login". This has something to do with Garena.
